packages/miniflare/src/plugins/assets/index.ts

```diff
@@ -135,6 +139,16 @@ export const ASSETS_PLUGIN: Plugin<typeof AssetsOptionsSchema> = {
 			);
 		}
 
+		if (staticRoutingContents !== undefined) {
+			const parsed = parseStaticRouting(staticRoutingContents);
+			logger.info(`✨ Parsed _routes.json.`);
+
+			options.assets.routerConfig = {
+				...options.assets.routerConfig,
+				static_routing: parsed,
+			};
+		}
+
```

Comment:
I think we also need to set the `has_static_routing` property in `assetConfig`: https://github.com/cloudflare/workers-sdk/pull/9280#diff-fe4258475216eecc324be32ac5142931de7783ae372429feba0c565a5919a0b8R23